### ISSUE TITLE
Simplify staging into a single repository

### ISSUE THREAD

**Issue by RubenVerborgh** [solid/solid.github.io#243]
As I found out in #242, two repositories are currently being used:
- https://github.com/solid/solid.github.io/ for staging
- https://github.com/solid/solidproject.org/ for production

What's difficult is that both repositories are out of sync. So not everything _can_ be edited in this repository, simply because some things aren't here yet.

Is there a technical reason for this split?

I propose to work in a single repository https://github.com/solid/solidproject.org/, and use different branches there for staging and production, with the default branch being staging. The staging site can then be published somewhere else via CI.

Thoughts?

**Comment by Vinnl** [comment]
The reason is mostly that JD set up both websites to be run from GitHub Pages, i.e. they started this one for the new version, and still had the other one for the live site. Working from a single repository would certainly be preferable. (Especially given that TWiS _is_ developed on the other repository...)

One thing we could do here that does not require us to coordinate with JD would be to set up a build pipeline that automatically pushes a specific branch of the other repo to this one to deploy it. When doing that, we should also make sure to transfer the issues and other project files from this repo to the main one.

The main reason this hasn't happened yet and as far as I know won't happen in the short term, is lack of resources.

**Comment by RubenVerborgh** [comment]
> Working from a single repository would certainly be preferable.

If the problem is just the CI pipeline, I will set that up. It takes under an hour.
From whom do I need approval?

**Comment by Vinnl** [comment]
It's both that and the transfer of non-repository content from this project to the other. (And updating the process document, I presume?)

I'd be very happy if you'd set it up. I don't think special approval is needed, @NSeydoux, @Mitzi-Laszlo?

**Comment by NSeydoux** [comment]
I don't think so, in my understanding the only critical operation requiring approval is the publication of content on solidproject.org. If you know how to set this up @RubenVerborgh it would certainly be a cleaner process, and all relevant discussion could happen in the solidproject.org repo. I guess you would need access rights set to an admin access to https://github.com/solid/solidproject.org during the process.

**Comment by RubenVerborgh** [comment]
Okay, I'm on it!
